packages/multichain-network-controller/src/types.ts

```diff
@@ -27,49 +27,52 @@ export type CommonNetworkConfiguration = {
    * EVM network flag.
    */
   isEvm: boolean;
-  /**
-   * The block explorers of the network.
-   */
-  blockExplorerUrls: string[];
-  /**
-   * The index of the default block explorer URL.
-   */
-  defaultBlockExplorerUrlIndex: number;
   /**
    * The chain ID of the network.
    */
-  chainId: SupportedCaipChainId;
+  chainId: CaipChainId;
   /**
    * The name of the network.
    */
   name: string;
-  /**
-   * The native asset type of the network.
-   */
-  nativeCurrency: CaipAssetType;
 };
 
 export type NonEvmNetworkConfiguration = CommonNetworkConfiguration & {
+  /**
+   * EVM network flag.
+   */
   isEvm: false;
+  /**
+   * The native asset type of the network.
+   */
+  nativeCurrency: CaipAssetType;
 };
 
 // TODO: The controller only supports non-EVM network configurations at the moment
 // Once we support Caip chain IDs for EVM networks, we can re-enable EVM network configurations
-// export type EvmNetworkConfiguration = CommonNetworkConfiguration & {
-//   isEvm: true;
-//   /**
-//    * The RPC endpoints of the network.
-//    */
-//   rpcEndpoints: string[];
-//   /**
-//    * The index of the default RPC endpoint.
-//    */
-//   defaultRpcEndpointIndex: number;
-// };
+export type EvmNetworkConfiguration = CommonNetworkConfiguration & {
+  /**
+   * EVM network flag.
+   */
+  isEvm: true;
+  /**
+   * The native asset type of the network.
+   * For EVM we use a string because there's no standard for tickers to be translated to Caip asset ID.
```

Comment:
Let's add a comment that this is a ticker

Reply:
Done